beacon_node/beacon_chain/src/chain_config.rs

```diff
@@ -67,6 +67,8 @@ pub struct ChainConfig {
     pub prepare_payload_lookahead: Duration,
     /// Use EL-free optimistic sync for the finalized part of the chain.
     pub optimistic_finalized_sync: bool,
+    /// Whether backfill sync rate-limiting is enabled.
```

Comment:
```suggestion
    /// Whether backfill sync processing should be rate-limited.
```
Small preference here since we have the concept of rate limiting sync _requests_ (to other peers) as well. This makes it clear we stagger processing, not requests